internal/csi-common/utils.go

```diff
@@ -193,12 +193,18 @@ func logGRPC(
 	handler grpc.UnaryHandler,
 ) (interface{}, error) {
 	log.ExtendedLog(ctx, "GRPC call: %s", info.FullMethod)
+	// TODO: remove the following check for next release
+	// refer to https://github.com/ceph/ceph-csi/issues/3314.
```

Comment:
why did TODO remove it in the next release? to support volume replication operator with cephcsi? or anything else?

Reply:
We will be removing volrep operator support in next release. 

So we dont need to check for replication secret type and directly strip it with csi sanitizer.